### PR TITLE
Redo profile page

### PR DESCRIPTION
**Pull request by ahsu1230**

### CODE REVIEW

File: winterfell/webapp/src/js/profileCtrl.js [hunk 1]
```diff
@@ -1,54 +1,48 @@
...
-      },
-      {
-        rank: 'captain',
-        branch: 'army',
-        yearStart: '2012',
-        yearEnd: '2013'
-      }
-    ];
+    $scope.user = Profile.user.user;
+    $scope.claims = []; // list of user's claims
```
Comment (qwwqwwq) [inline winterfell/webapp/src/js/profileCtrl.js]: You can resolve claims through the UI router and then inject them into the controller, similar to how we do for other things like the user and forms, etc.

File: winterfell/webapp/src/templates/profile/claims.jade [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
-                    div.vfi-icon.vfi-view
-        div.vfi-check-icon(ng-if="claim.state=='submitted' || claim.state=='processed'")
-        div.vfi-bar
-            div.vfi-progress(ng-class="{'vfi-incomplete': claim.state=='incomplete', 'vfi-submitted': claim.state=='submitted', 'vfi-processed': claim.state=='processed'}")
+div.profile-section#claims
+
+  div.no-claims-wrapper(ng-show="claims.length == 0")
+    h4 You have no pending file claims at the moment.
+    br
+    p Click #[a(ui-sref="root.claimstart") here] to start a file claim.
```
Comment (qwwqwwq) [inline winterfell/webapp/src/templates/profile/claims.jade]: so in jade/pug the `[` denoes an inline HTML element?

Comment (ahsu1230) [inline winterfell/webapp/src/templates/profile/claims.jade]: yup! http://stackoverflow.com/questions/6989653/jade-links-inside-a-paragraph
shows you how to do inline elements

File: winterfell/webapp/src/js/profileCtrl.js (hunk 1 above)
Comment (ahsu1230) [inline winterfell/webapp/src/js/profileCtrl.js]: What's the difference between doing that and using `claimService.userClaims` instead?

**Pull request merged**